packages/dai/src/eth/TransactionManager.js

```diff
@@ -196,13 +196,16 @@ export default class TransactionManager extends PublicService {
 
   async _getGasLimit(options, contract, method, args) {
     let transaction = {};
-    let data = contract.interface.functions[method](...args).data;
+    let data = contract.interface.encodeFunctionData(method, args);
```

Comment:
The old way to get contract function data was deprecated, now we use `encodeFunctionData`